### PR TITLE
Feature#25 fixing UI

### LINKED ISSUE #25: Fixing UI
# REQUIREMENTS
Nice and friendly interfaces. 

# Acceptance Criteria
List all required acceptance criteria
- [ ] UI easy to learn
- [ ] UI easy to use

### PR DESCRIPTION
**Pull request by mariam-rivera**
# Description
Some UI screens were updated for a better user experience. Also, unnecessary code was removed. 

 Examples:
 - Some larges JSONs were separated in new files. 
 - Files that we don't use were deleted. 
 - Lines with console.log(x) or imports that we don't use were deleted too.
 
  ## Screenshots 
- Here. This is the home screen where the user can navigate through the application. 

- Some COVID-19 instructions and preventions were added.

- The user can add all the tests that he has made and have a list test like the following example.

### CODE REVIEW

File: COVID19/src/screens/registerSymptoms.js [hunk 1]
```diff
@@ -27,8 +25,6 @@ const RegisterSymptoms = ({navigation, route}) => {
         url: 'https://hidden-cliffs-21927.herokuapp.com/api/incident',
```
Comment (fernandoamz) [inline COVID19/src/screens/registerSymptoms.js]: The API url should be as constant

File: COVID19/src/screens/Prevention.js [hunk 2]
```diff
@@ -12,33 +13,58 @@ const Prevention = ({navigation}) => {
...
           title: constans.keepDistance,
+          subtitle: constans.keepDistanceSub,
         },
         {
           backgroundColor: colors.purpleMedium,
-          image: <Image source={require('../utils/images/sanitizer.png')} />,
+          image: (
+            <Image
+              style={styles.imageSize}
+              source={require('../utils/images/sanitizer.png')}
```
Comment (fernandoamz) [inline COVID19/src/screens/Prevention.js]: Try to import all the images at the file's beginning

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: COVID19/src/screens/Prevention.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ import {Image} from 'react-native';
 import Onboarding from 'react-native-onboarding-swiper';
 import colors from '../utils/colors';
 import constans from '../utils/constans';
+import styles from '../utils/styles/preventionStyles/preventionStyles';
```
Comment (fernandoamz) [inline COVID19/src/screens/Prevention.js:12]:
```suggestion
import maskImageCarrousel from '../utils/images/mask.png';
import handsImageCarrousel from '../utils/images/hands.png';
import distanceImageCarrousel from '../utils/images/distance.png';

```

File: COVID19/src/screens/Prevention.js [hunk 4]
```diff
@@ -12,33 +13,58 @@ const Prevention = ({navigation}) => {
       pages={[
         {
           backgroundColor: colors.purpleMedium,
-          image: <Image source={require('../utils/images/mask.png')} />,
+          image: (
+            <Image
+              style={styles.imageSize}
+              source={require('../utils/images/mask.png')}
```
Comment (fernandoamz) [inline COVID19/src/screens/Prevention.js]:
```suggestion
              source={maskImageCarrousel}
```

File: COVID19/src/screens/Prevention.js [hunk 5]
```diff
@@ -12,33 +13,58 @@ const Prevention = ({navigation}) => {
...
           title: constans.wearMask,
+          subtitle: constans.wearMaskSub,
         },
         {
           backgroundColor: colors.purpleLight,
-          image: <Image source={require('../utils/images/hands.png')} />,
+          image: (
+            <Image
+              style={styles.imageSize}
+              source={require('../utils/images/hands.png')}
```
Comment (fernandoamz) [inline COVID19/src/screens/Prevention.js]:
```suggestion
              source={handsImageCarrousel}
```

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: COVID19/src/screens/Prevention.js (hunk 2 above)
Comment (hearagmu) [reply COVID19/src/screens/Prevention.js]: fixed

File: COVID19/src/screens/Prevention.js (hunk 3 above)
Comment (hearagmu) [reply COVID19/src/screens/Prevention.js:12]: fixed

File: COVID19/src/screens/Prevention.js (hunk 4 above)
Comment (hearagmu) [reply COVID19/src/screens/Prevention.js]: Good tip, I did not know

File: COVID19/src/screens/Prevention.js (hunk 5 above)
Comment (hearagmu) [reply COVID19/src/screens/Prevention.js]: fixed

**Review by fernandoamz — APPROVED** [review]

**Review by ChantalRamirez — APPROVED** [review]

**Review by HRuvalcabaP — APPROVED** [review]
Good job! :3

**Pull request merged**